cdbot/cogs/admin.py

```diff
@@ -0,0 +1,51 @@
+import re
+
+from cbot.constants import NICKNAME_PATTERNS, PLACEHOLDER_NICKNAME
+from discord.ext.commands import Bot, Cog
+
+
+def checkName(nick):
```

Comment:
Maybe replace with a list comprehension?
```suggestion
def checkName(nick):
    return any(re.match(i,nick) for i in NICKNAME_PATTERNS)
```

Reply:
Or alternatively, a more efficient for loop:
```suggestion
def checkName(nick):
    for i in NICKNAME_PATTERNS:
         if re.match(i,nick):
             return True
    return False
```